### PR TITLE
Replace the Logo with Meshery in config.yml

### PR DESCRIPTION
**Pull request by Simar-malhotra09**
**Description**

 Replaced  the Layer5 logo with Meshery in the config.yml workflow file and to account for dark and light theme.

**Notes for Reviewers**
Please review the changes to ensure that the Meshery logo has been correctly added to the config.yml workflow file and that there are no unintended consequences of this change. Additionally, please let me know if there are any other places where the Layer5 logo needs to be updated to Meshery.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../img/meshery_dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="../img/meshery_light.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
```
Comment (leecalcote) [inline .github/config.yml:23]: Thanks, @Simar-malhotra09. This source needs to be changed. // @goldfishdolphin

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by thisiskaransgit** [comment]
@Simar-malhotra09, what's your progress, if you are stuck somewhere discuss it on slack, and also fix the merge conflicts

**Comment by Simar-malhotra09** [comment]
@thisiskaransgit hey, sorry for some reason i did not get the notification regarding the changes requested. I unforseably had to move countries, so i couldn't find the time to check in either. This change should be simple enough, i'll get to it asap

File: .github/config.yml [hunk 2]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
```
Comment (thisiskaransgit) [inline .github/config.yml:18]:
```suggestion
  Thanks for your contribution to Meshery! :tada:
```
And, 
We recommend using uniform name conventions for images throughout the project, update the names to
Meshery-celebration-Dark.svg
Meshery-celebration-Light.svg

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: .github/config.yml [hunk 3]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
...
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../img/meshery_dark.svg">
```
Comment (goldfishdolphin) [inline .github/config.yml:21]:
```suggestion
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-linked/master/.github/welcome/meshery_dark.svg">
```

**Review by goldfishdolphin — COMMENTED** [review]
Please change the source of images to .github folder and update the path.

**Comment by thisiskaransgit** [comment]
@Simar-malhotra09, are you working on it, if not signal please signal

**Comment by thisiskaransgit** [comment]
@Ghat0tkach, will you work on this one? Opening up a new PR and linking this one will do the work.

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach, will you work on this one? Opening up a new PR and linking this one will do the work.

Yup sure , Thank you!

**Pull request closed without merging**